.gitlab-ci-check-docker-deploy.yml

```diff
@@ -0,0 +1,96 @@
+# .gitlab-ci-check-docker-deploy.yml
+#
+# This gitlab-ci template modifies a manifest file in an infrastructure
+# repository to trigger de deploy of the newly build Docker image.
+#
+# It depends on .gitlab-ci-check-docker-build.yml, so both need to be
+# included for the process to work.
+#
+#
+# Add it to the project in hand through Gitlab's include functionality
+#
+# include:
+#   - project: 'Northern.tech/Mender/mendertesting'
+#     file: '.gitlab-ci-check-docker-deploy.yml'
+#
+# Requires .gitlab-ci-check-docker-build.yml to be included and related
+# varibales to be set.
+#
+# Requires TARGET_MANIFEST_FILE to be set in the calling Pipeline. It
+# can be one or more, separated by commas.
+#
+# The job will modify for the manifest file(s) in sre-repo, master branch,
+# by default. It can overriden with INFRA_REPOSITORY and INFRA_BRANCH
+# variables accordingly
+#
+# variables:
+#   TARGET_MANIFEST_FILE: <path to yaml file>,<path to other yaml file>
+#   INFRA_REPOSITORY: <git@github.com:mendersoftware/other-repo.git> (optional)
+#   INFRA_BRANCH: <other-branch> (optional)
+#
+# By default, it only runs on master branch, override only tag to modify
+# behaviour, for example with a regex:
+#
+# only:
+#   refs:
+#     - /^(master|staging|production|feature-.+|[0-9]+\.[0-9]+\.x)$/
+#
+
+stages:
+  - build
+  - publish
+  - sync
+
+include:
+- project: 'Northern.tech/Mender/mendertesting'
+  file: '.gitlab-ci-check-docker-build.yml'
+
+variables:
+  INFRA_REPOSITORY: "git@github.com:mendersoftware/sre-tools.git"
+  INFRA_BRANCH: "master"
```

Comment:
Should probably include `DEBIAN_FRONTEND: noninteractive`.

Reply:
:+1: